test/models/concerns/audit_extensions_test.rb

```diff
@@ -41,4 +41,9 @@ def setup
     FactoryGirl.create :lookup_value, :lookup_key_id => key.id, :value => false, :match => "hostgroup=Common"
     refute_empty Audit.search_for("type = lookupvalue")
   end
+
+  test "search for type=compute_resource in audit" do
+    FactoryGirl.create(:ec2_cr)
+    refute_empty Audit.search_for("type = compute_resource")
```

Comment:
also would be nice to show that you search for different compute resource types ?(e.g. that STI works correctly).